### ISSUE TITLE
duplicate games and rooms

### ISSUE THREAD

**Issue by alee** [virtualcommons/port-of-mars#364]
Duplicate rooms and games are being created for some reason on the server side + backend. @cpritcha can you look into this?

```
 230 | cl9m7EKvg | 2020-03-26 01:34:00.034426 |                 1 | f
 231 | ubobXqhF6 | 2020-03-26 01:34:00.047505 |                 1 | f
 232 | 2UHhpwuVz | 2020-03-26 01:34:00.068379 |                 1 | f
 233 | ioEPk4eey | 2020-03-26 01:35:00.027789 |                 1 | f
 234 | GTuPvHA8p | 2020-03-26 01:35:00.038299 |                 1 | f
 235 | 34weW1IV4 | 2020-03-26 01:35:00.046376 |                 1 | f
 236 | KBSpZsVyy | 2020-03-26 01:36:00.046198 |                 1 | f
 237 | 9FFDLJ7fE | 2020-03-26 01:36:00.066422 |                 1 | f
 238 | BkP2HuhPF | 2020-03-26 01:36:00.078553 |                 1 | f
 239 | gX-7pwl_9 | 2020-03-26 01:37:00.037752 |                 1 | f
 240 | 8QjYa2yaK | 2020-03-26 01:37:00.030616 |                 1 | f
 241 | wqvD-xk04 | 2020-03-26 01:37:00.043064 |                 1 | f
 242 | jaA1UXjR5 | 2020-03-26 01:38:00.028314 |                 1 | f
 243 | lJPfm_vlr | 2020-03-26 01:38:00.038812 |                 1 | f
 244 | nE0pHfCPz | 2020-03-26 01:38:00.049381 |                 1 | f
 245 | AD7HzpNjc | 2020-03-26 01:39:00.030755 |                 1 | f
 246 | 97Pd4-pZ4 | 2020-03-26 01:39:00.041886 |                 1 | f
 247 | rV-3vTa5Z | 2020-03-26 01:39:00.046713 |                 1 | f
```

**Comment by alee** [comment]
this might have been fixed via c23f8c4e37134730cfb4b20752054f7dfb09867d

**Comment by alee** [comment]
This appears to have been fixed after redistributeGroups no longer eagerly creates empty groups.
